internal/service/ec2/find.go

```diff
@@ -447,7 +447,23 @@ func FindRouteTableByID(conn *ec2.EC2, routeTableID string) (*ec2.RouteTable, er
 }
 
 func FindRouteTable(conn *ec2.EC2, input *ec2.DescribeRouteTablesInput) (*ec2.RouteTable, error) {
```

Comment:
For an alternative pattern take a look at `FindInternetGateway`/`FindInternetGateways` - The singular calls the plural and adds more checks.

Reply:
Ooo much nicer with the return errors -- will make updates here 👍